LoopTestingKit/TestingScenarioInstance.swift

```diff
@@ -8,12 +8,16 @@
 
 import LoopKit
 
+public enum InjectedAction: String {
```

Comment:
Would recommend that the protocol not encode the specific actions possible; the range of possible actions for a given device is the domain of the testing device itself, not global to all testing devices.

Reply:
That makes sense. I originally did this for compiler support when it came to exhaustivity of these actions; i.e. when a new action is added, we'll have to determine if we need support for it in each testing device. This is a good suggestion though because I think you're right that not all testing devices should be forced to implement a new action unless it needs to handle a specific action. I'll update this!